src/SFA.DAS.ApprenticeFeedback.Application/Commands/CreateApprenticeFeedback/CreateApprenticeFeedbackHandler.cs

```diff
@@ -0,0 +1,63 @@
+﻿using MediatR;
+using SFA.DAS.ApprenticeFeedback.Domain.Interfaces;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Linq;
+
+namespace SFA.DAS.ApprenticeFeedback.Application.Commands.CreateApprenticeFeedback
+{
+    public class CreateApprenticeFeedbackHandler : IRequestHandler<CreateApprenticeFeedbackCommand, CreateApprenticeFeedbackResponse>
+    {
+        public readonly IApprenticeFeedbackRepository _apprenticeFeedbackRepository;
+
+        public CreateApprenticeFeedbackHandler(IApprenticeFeedbackRepository apprenticeFeedbackRepository)
```

Comment:
Worth adding in a logger injected here to then put some logging calls in, usually an entry and exit call "Create apprentice feedback for target:id" and "successfully created feedback with id" with the returned ID from the database call. usually at Debug or Info level